cengage/react-magma
# fix(Indeterminate Checkbox): Improved accessibility for `Indeterminate status (#1231)

Issue: #1231

## What I did
Added aria-checked="mixed" to the Indeterminate checkbox when in the indeterminate status

## Screenshots:


## Checklist 

## How to test
Open the storybook the Indeterminate Checkbox section
Confirm new aria-checked property for Indeterminate checkbox
